redesign/dist/result/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="/styles/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i,900,900i" rel="stylesheet">
+    <title>Result | Expunge Maryland</title>
+</head>
+
+<body>
+
+
+<div class= "nav-wrapper">
+	<nav class= "navigation">	
+			<button id="js-menu-button" aria-controls="js-menu" aria-expanded="false" class="u-button-reset menu--button">
+			<span aria-hidden="true" class="menu-icon">
+				 <div class="menu-icon--line"></div>
+				 <div class="menu-icon--line"></div>
+				 <div class="menu-icon--line"></div>
+			</span>
+		</button>
+		
+		<ul id="js-menu" class="u-unstyled-list menu--list" aria-hidden="false" data-visible="false" >
+			<li class="menu--text"><a href="#" target="_self" class="u-block link" >01 . What is expungement?</a></li>
+			<li class="menu--text"><a href="/disclaimer" target="_self" class="u-block link">02 . Can I expunge my record?</a></li>
+			<li class="menu--text"><a href="/resources" target="_self" class="u-block link">03 . Resources</a></li>
+			<li class="menu--text"><a href="/faq" target="_self" class="u-block link">04 . FAQs</a></li>
+			<li class="menu--text"><a href="#" target="_self" class="u-block link">05 . About us</a></li>
+		</ul>
+	</nav>
+</div>	
+
+<header class="sticky image">
+    <a href="/index.html"><img class="logo--expunge" src="/images/logo--expunge.svg" alt="Expunge Maryland logo."></a>
+</header>
+
+
+<section class= "site-wrapper">
+    <section class="main">
+
+        <h1 class="text--header u-center u-bold u-add-margin">Great!</h1>
+        <section class="question result">
+            <p class="disclaimer--text u-remove-margin result">Based on your answers you’ve provided, you can likely expunge your case!</p>
+            <p class="disclaimer--text u-remove-margin result">We have partnered with the Maryland Volunteer Lawyers Service (MVLS) to help you on your way toward expungement. Click the button below to begin.</p>
+        </section>
+
+        <section class="answers">
+            <a class= "answer" target="_blank" href="https://mvlslaw.org/">Start the Process with MLVS</a>
```

Comment:
Is delivering users to their homepage the best option? What about something like this: https://mvlslaw.org/get-legal-help/ instead? Just wondering what would be the least jarring and most respectful option, given that they've already put some time in on this process and you wouldn't want them to feel like they are starting from scratch with a new org.